### DISCUSSION TITLE
Any problems with auto-updates to DSM?

### DISCUSSION THREAD

**Discussion by signal15** [bb-qq/r8152#399] [Q&A] [answered]
I have the Asus adapter and have it working.  However, I have auto-updates enabled, and I don't want to lose access to my NAS.  Has anyone had issues where an auto-update caused problems with this driver?

**Comment by bb-qq** [comment, marked as answer]
Driver compatibility will be lost during major/minor version upgrades, but it will not be lost during micro version upgrades.

I do not know the behavior of the auto-update of DSM, I recommend disabling it.

**Reply by bb-qq** [reply]
I think I have answered your question, so I will close this thread. If you have any questions, feel free to re-open it.
